zebra-state/src/request.rs

```diff
@@ -264,19 +265,31 @@ pub enum Request {
     /// [`block::Height`] using `.into()`.
     Block(HashOrHeight),
 
-    /// Request a UTXO identified by the given Outpoint, waiting until it becomes
-    /// available if it is unknown.
+    /// Requests a spendable [`Utxo`] identified by the given Outpoint.
+    /// If the UTXO is unknown or unspendable, waits until it becomes available
+    /// and spendable.
     ///
     /// This request is purely informational, and there are no guarantees about
     /// whether the UTXO remains unspent or is on the best chain, or any chain.
     /// Its purpose is to allow asynchronous script verification.
     ///
+    /// However, the spendability of a valid UTXO should not change, because
+    /// [coinbase transaction IDs commit to the relevant data](https://github.com/ZcashFoundation/zebra/blob/main/book/src/dev/rfcs/0004-asynchronous-script-verification.md#parallel-coinbase-checks).
```

Comment:
The U in UTXO stands for Unspendable, so "spendable UTXO" is a misnomer. I think this meant to say "transaction output" (twice). Also, I don't understand the last sentence; of course an unspent transaction output can potentially be spent (or the chain rolled back to a point before a spent transaction output was spent).

Reply:
I agree that this wording is confusing, and so is the design. We're thinking about going with a simpler design. (And reverting this part of the change.)

The last sentence is meant to refer to the coinbase maturity and coinbase transparent output shielding consensus rules.

Reply:
This change was reverted as part of a rebase and force-push.